cfme/tests/automate/test_method.py

```diff
@@ -749,3 +750,50 @@ def test_null_coalescing_fields(request, klass):
     )
 
     assert log.validate()
+
+
+@pytest.mark.tier(3)
+def test_automate_user_has_groups(request, appliance, custom_instance):
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1411424
```

Comment:
**Suggestion:**
If you are mentioning Bugzilla under tag, I don't think in the description it is necessary.